common/sgx/tcbinfo.h

```diff
@@ -24,13 +24,13 @@ typedef union _oe_tcb_level_status {
         uint32_t outofdate : 1;            //! "OutOfDate"
         uint32_t configuration_needed : 1; //! "ConfigurationNeeded"
         uint32_t up_to_date : 1;           //! "UpToDate"
-
         /*! "OutOfDateConfigurationNeeded"
          *
          * This tcb status indicates that the QE Identity Info is out of date
          * and the TCB Info requires configuration "ConfigurationNeeded"
          */
         uint32_t qe_identity_out_of_date : 1;
+        uint32_t sw_hardening_needed : 1; //! "SWHardeningNeeded"
```

Comment:
Minor: maybe consider align this comment with other comment above at column 44 (full 4-space tabs), this one is at column 43.